numpyro/hmc_util.py

```diff
@@ -561,8 +561,8 @@ def _potential_energy(params):
     return _potential_energy
 
 
-def transform_fn(transforms, params, invert=False):
-    return {k: transforms[k](v) if not invert else transforms[k].inv(v)
+def transform_fn(inv_transforms, params, unconstrained=True):
+    return {k: inv_transforms[k](v) if unconstrained else inv_transforms[k].inv(v)
```

Comment:
Maybe we can remove `unconstrained` kwargs, hidden this `transform_fn`, and make it only do inverse transform? Kind of
```
def _inv_transform_fn(inv_transforms, params):
    return {k: inv_transforms[k][v] for k, v in params.items()}
```